client/src/components/map/controls/basemap/component.tsx

```diff
@@ -0,0 +1,47 @@
+import Tooltip from 'components/tooltip';
+import Image from 'next/image';
+import { useCallback } from 'react';
+
+import SatelliteImage from './images/satellite1.png';
+import DefaultImage from './images/default1.png';
+
+export type BasemapValue = 'satellite' | 'default';
+
+interface BasemapControlProps {
+  value: BasemapValue;
+  onChange: (value: BasemapValue) => void;
+}
+
+const BasemapControl: React.FC<BasemapControlProps> = ({ value, onChange }) => {
+  const handleClick = useCallback(() => {
+    if (value === 'default') {
+      onChange('satellite');
+    } else {
+      onChange('default');
+    }
+  }, [onChange, value]);
+
+  const image = value === 'default' ? SatelliteImage : DefaultImage;
+
+  return (
+    <Tooltip
+      hoverTrigger
+      placement="right"
+      theme="dark"
+      content={<div className="bg-gray-900 text-white rounded-lg p-2">Change basemap!</div>}
```

Comment:
```suggestion
      content={<div className="bg-gray-900 text-white rounded-lg p-2">Change basemap</div>}
```

Reply:
Removed it, also added the name of the basemap that will be changed to on swap